setup.py

```diff
@@ -23,7 +23,6 @@
     include_package_data=True,
     zip_safe=False,
     dependency_links=[],
-    install_requires=requirements,
```

Comment:
i think this line is needed

Reply:
hmm, this must have got deleted on accident. Thanks for catching it!!